server/routes/slack/slack_events.py

```
@@ -556,4 +589,208 @@ def _handle_suggestion_details(payload: dict, action: dict, slack_user_id: str,
+            workspace_user_id = get_user_id_from_slack_team(team_id)
+            if workspace_user_id:
+                try:
+                    workspace_token_data = get_user_token_data(workspace_user_id, "slack")
+                    if workspace_token_data:
+                        _send_ephemeral(
+                            SlackClient(workspace_token_data.get('access_token')),
+                            channel_id, slack_user_id,
+                            f"WARNING: You're not authenticated in Aurora.\n\nTo dismiss recommendations, connect your Aurora account:\n{FRONTEND_URL}/settings/integrations\n\nClick 'Connect' for Slack and authorize this workspace.",
+                        )
+                except Exception as e:
+                    logger.exception(f"Failed to warn unauthenticated Slack user {sanitize(slack_user_id)}: {e}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)